src/Common/ColumnsHashingImpl.h

```diff
@@ -125,7 +125,7 @@ class FindResultImpl : public FindResultImplBase, public FindResultImplOffsetBas
 
 public:
     FindResultImpl()
-        : FindResultImplBase(false), FindResultImplOffsetBase<need_offset>(0)
+        : FindResultImplBase(false), FindResultImplOffsetBase<need_offset>(0), value(nullptr)
```

Comment:
We intentionally allow uninitialized value here.

Reply:
a comment is added instead.